src/playback/playback.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import DashboardPage from '../utils/DashboardPage';
+import PlaybackSummary from './summary';
+import Section from '../utils/Section';
+
+export default class Playback extends React.Component {
+  render() {
+    const sections = [
+      {
+        title: 'Firefox (Desktop)',
+        bitsVersions: [64, 32],
+        encodings: ['VP9', 'H264'],
+        browserId: 'firefox',
+      }, {
+        title: 'Firefox (Android)',
+        bitsVersions: [64],
+        encodings: ['VP9', 'H264'],
+        browserId: 'geckoview',
+      }, {
+        title: 'Firefox Preview (Android)',
+        bitsVersions: [64],
+        encodings: ['VP9', 'H264'],
+        browserId: 'fenix',
+      },
+    ];
+
+    return (
+      <DashboardPage
+        title="Playback"
+        subtitle="Release criteria"
+      >
+        {sections.map((
+          {
+            title, bitsVersions, encodings, browserId,
+          },
+        ) => (
+          <Section title={title} key={title}>
+            <Grid container spacing={2}>
+              {bitsVersions.map(bits => (
```

Comment:
This one is tough, since there is only one 32 bit platform.  I suggest instead of 'bitsVersions.map()', do not `map()` at all: Instead feed all bits into `<PlaybackSummary`, like so,

                    <PlaybackSummary
                      key={`ps_${encoding}_${browserId}`}
                      bits={[32, 64]}
                      encoding={encoding}
                      browserId={browserId}
                    />